wikimedia/wikipedia-ios
# Handle Xcode 13 warnings, reintroduce peek/pop

**Phabricator:** https://phabricator.wikimedia.org/T289535

### Notes
* On all peeks except those from within an article, the preview VC is slightly too large for a split second, then comes back to normal. It seems better - but not perfect - w/o any actions. This problem didn't exist in old version of peek, but that was presumably implemented very differently by Apple. I'm open to suggestions on how to fix this on our end, if it seems like a major issue.
* There are warnings when running tests - I'll fix those in a separate PR.
* I introduced and fixed a bug where announcement/fundraiser is presented over a pop - borking UI, requiring hard quit.
* I snuck in a small fix - when in the Places list view and selecting "see on map" from context menu, the segmented control showed the wrong state.

### Test Steps
1. Run app, ensure it shows no warnings.
1. Peek and pop everywhere - explore menu, Saved, Places list view, History, articles, everywhere.
1. In general, spend a little bit with this to make sure there aren't any regressions.
